From bc68683e1ed1579718291a74edc7c6406ce078a0 Mon Sep 17 00:00:00 2001
From: Daan De Meyer <daan.j.demeyer@gmail.com>
Date: Mon, 31 Jul 2023 20:58:31 +0200
Subject: [PATCH] kernel-install: Make sure KERNEL_INSTALL_BYPASS is disabled
 in tests

(cherry picked from commit 4435da1f1732e2078e42b0ee43ad56fde2b021a3)

Related: RHEL-27512
---
 src/kernel-install/test-kernel-install.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/kernel-install/test-kernel-install.sh b/src/kernel-install/test-kernel-install.sh
index 2e44063668..3b9ff76e60 100755
--- a/src/kernel-install/test-kernel-install.sh
+++ b/src/kernel-install/test-kernel-install.sh
@@ -32,6 +32,7 @@ export KERNEL_INSTALL_CONF_ROOT="$D/sources"
 export KERNEL_INSTALL_PLUGINS="$plugin"
 export BOOT_ROOT="$D/boot"
 export MACHINE_ID='3e0484f3634a418b8e6a39e8828b03e3'
+export KERNEL_INSTALL_BYPASS="no"
 
 "$kernel_install" -v add 1.1.1 "$D/sources/linux" "$D/sources/initrd"
 
